src/minimodem.c

```diff
@@ -664,6 +666,16 @@ main( int argc, char*argv[] )
 	    case 'f':
 			filename = optarg;
 			break;
+	    case 'O':
+#if USE_STDIO
+			sa_backend = SA_BACKEND_STDIO;
+                        // Apparently the receiver wants floats, so I'mma just output floats
+                        sample_format = SA_SAMPLE_FORMAT_FLOAT;
```

Comment:
There's a `--float-samples` command line flag. Instead of forcing the value here, couldn't you rely on that flag?